run_aprime.bash

```diff
@@ -353,35 +343,48 @@ if [ ! -d $www_dir/$plots_dir_name ]; then
 else
   rm -f $www_dir/$plots_dir_name/*
 fi
+# Check on MPAS-Analysis www_dir, permissions included
+# Create it if it does not exist, purge it if it does
+export mpas_www_link=mpas-analysis # mpas_www_link is used by generate_html to define
+                                   # the html link to the MPAS-Analysis web interface
+if [ ! -d $www_dir/$mpas_www_link ]; then
+  mkdir -p $www_dir/$mpas_www_link
+  chmod -R ga+rX $www_dir/$mpas_www_link
+fi
+export mpas_www_link=$mpas_www_link/$plots_dir_name
+export mpas_www_dir=$www_dir/$mpas_www_link
+if [ ! -d $mpas_www_dir ]; then
+  mkdir -p $mpas_www_dir
+else
+  rm -rf $mpas_www_dir/*
+fi
 
 # LOAD THE MACHINE-SPECIFIC ANACONDA-2.7 ENVIRONMENT
 source $MODULESHOME/init/bash  
 if [ $machname == "nersc" ]; then
   module unload python
   module unload python_base
   module use /global/project/projectdirs/acme/software/modulefiles/all
-  module load python/anaconda-2.7-acme
-  export NCO_PATH_OVERRIDE=No
+  module load e3sm-unified/1.1.3
```

Comment:
Wow, a-prime is more out of date than I realized!  I had to delete python/anaconda-2.7-acme to make space in the acme project so a-prime has been broken on NERSC for a couple of weeks.  No one said anything to me, which is kind of worrisome!